### PR TITLE
Feature/external iexec oci

### PR DESCRIPTION
**Pull request by jeremyjams**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
-@Library('global-jenkins-library@1.3.0') _
+@Library('global-jenkins-library@1.8.0') _
 buildJavaProject(
+        buildInfo: getBuildInfo(),
         integrationTestsEnvVars: [],
         shouldPublishJars: true,
         shouldPublishDockerImages: true,
         dockerfileDir: '.',
-        //dockerfileFilename: "Dockerfile-local",
-        buildContext: '.',
```
Comment (jbern0rd) [inline Jenkinsfile:8]: why did you remove this configuration which is kept in the other PRs ?

Comment (jeremyjams) [reply Jenkinsfile:8]: Current implementation of jenkins-library is awkward: If `buidContext` is not set, default `buildContext` will be `dockerfileDir` (whose default is `docker/`). In next PRs, it might be best to remove that rule and set all defaults to `.`.

Comment (jbern0rd) [reply Jenkinsfile:8]: I agree I find this part awkward too and to work on this in a next development.

**Review by jbern0rd — APPROVED** [review]
